src/middleware.ts

```diff
@@ -103,7 +103,7 @@ export const devtools = <S extends State>(
           message.payload.nextLiftedState?.computedStates || []
 
         computedStates.forEach(
-          ({ state }: { state: PartialState<S> }, index: number) => {
+          ({ state }: { state: PartialState<S, keyof S> }, index: number) => {
```

Comment:
Just curious why this change is required.

Reply:
already resolved.

Reply:
FYI, it was leftover from my experiment with removing the default value for the second generic.